### PR TITLE
Made detector string configurable. Only calibration has been tested

### PR DESCRIPTION
**Pull request by valmar**

### CODE REVIEW

File: calibration/facility_specifics/cfel/generate_paths.py [hunk 1]
```diff
@@ -51,6 +52,7 @@ def __init__(self,
         self._asic = asic
         self._runs = runs
         self._run_name = run_name
+        self._detector_string = detector_sting
```
Comment (MatroseBlaubaer) [inline calibration/facility_specifics/cfel/generate_paths.py]: typo

File: calibration/facility_specifics/cfel/generate_paths.py [hunk 2]
```diff
@@ -37,7 +37,8 @@ def __init__(self,
                  meas_in,
                  asic,
                  runs,
-                 run_name):
+                 run_name,
+                 detector_sting):
```
Comment (MatroseBlaubaer) [inline calibration/facility_specifics/cfel/generate_paths.py]: typo

Comment (MatroseBlaubaer) [inline calibration/facility_specifics/cfel/generate_paths.py]: typo

File: calibration/facility_specifics/xfel/layout/xfel_layout_2017.py [hunk 3]
```diff
@@ -53,16 +54,17 @@ def __init__(self,
...
-            'cellid': ("INSTRUMENT/SPB_DET_AGIPD1M-1/DET/{}CH0:xtdf/"
-                       "image/cellId"),
+            'status': "INDEX/{}/DET/{{}}CH0:xtdf/image/status".format(self._detector_string),
+            'image_first': ("INDEX/{}/DET/{{}}CH0:xtdf/"
+                            "image/first".format(self._detector_string)),
+            'image_last': "INDEX/{}/DET/{{}}CH0:xtdf/image/last".format(self._detector_string),
+
+            'data': "INSTRUMENT/{}/DET/{{}}CH0:xtdf/image/data".format(self._detector_string),
+            'cellid': ("INSTRUMENT/{}/DET/{{}}CH0:xtdf/"
+                       "image/cellId".format(self_detector_string)),
```
Comment (MatroseBlaubaer) [inline calibration/facility_specifics/xfel/layout/xfel_layout_2017.py]: typo (missing .)?

**Comment by MatroseBlaubaer** [comment]
Hi Valerio,
did you fix these typos by now? It appears they are still here. Or maybe I just don't know how to see the update properly here, in which case I apologize
Stephan

**Comment by valmar** [comment]
No sorry I forgot about that. I fixed the typos now, please see the 
update in the pull request!

Valerio

**Pull request merged**